app/models/volunteer.rb

```diff
@@ -55,14 +60,14 @@ def active_for_authentication?
   end
 
   def inactive_message
-    if not assigned
+    if !assigned?
```

Comment:
Nice catch. Still, I prefer clarity over matching other docs, but it's up to you 👍